### PR TITLE
[PTDT-1552] Add LLM media types

### PR DESCRIPTION
**Pull request by lgluszek**
Jira issue: [Can't fetch a list of projects in SDK when a list contains an LLM project](https://labelbox.atlassian.net/browse/PTDT-1552)

How to test:
1. Add an LLM project
2. Run this code

```
projects = client.get_projects()
for project in projects:
    print(project.name)
```

3. Check if it works

image

### CODE REVIEW

File: labelbox/schema/media_type.py [hunk 1]
```diff
@@ -18,6 +18,8 @@ class MediaType(Enum):
     Video = "VIDEO"
     Unknown = "UNKNOWN"
     Unsupported = "UNSUPPORTED"
+    LLMPromptCreation = "LLM_PROMPT_CREATION"
```
Comment (kkim-labelbox) [inline labelbox/schema/media_type.py:21]: Could you order these alphabetically in the file?

File: labelbox/schema/media_type.py [hunk 2]
```diff
@@ -41,8 +43,10 @@ def is_supported(cls, value):
     @classmethod
     def get_supported_members(cls):
         return [
-            item for item in cls.__members__
-            if item not in ["Unknown", "Unsupported"]
+            item for item in cls.__members__ if item not in [
```
Comment (kkim-labelbox) [inline labelbox/schema/media_type.py:46]: Do we need to add LLM types as unsupported types here? Shouldn't adding the media types make `get_projects()` work with LLM projects?

Comment (lgluszek) [reply labelbox/schema/media_type.py:46]: It will work without it, but tests started failing.

Comment (lgluszek) [reply labelbox/schema/media_type.py:46]: @pd-lb do you remember which test started failing?

**Review by kkim-labelbox — APPROVED** [review]

**Pull request merged**